test/output/pod-with-emptydir.yaml

```diff
@@ -3,7 +3,7 @@ kind: Pod
 metadata:
   name: pod-with-emptydir
   annotations:
-    cluster-autoscaler.kubernetes.io/safe-to-evict: "true"
+    cluster-autoscaler.kubernetes.io/safe-to-evict: true
```

Comment:
The quote for `true` has to be there otherwise the annotation patch would be ignored while applying it. As here annotation takes map[string]string and true is a special string in this case so a quote will explicitly say treat it as a string rather than a boolean.

Reply:
Let's understand more of these types are interpreted, as it's confusing when to use quotes and when not to even though the final type is a string.